src/project/internal/projectmigrator.cpp

```diff
@@ -142,6 +142,7 @@ Ret ProjectMigrator::migrateProject(engraving::EngravingProjectPtr project, cons
         return make_ret(Ret::Code::InternalError);
     }
 
+    score->lockUpdates(false);
```

Comment:
We need to make sure that this change won't cause performance issues when opening old scores (because before we called score->update() a lot of times and this caused a big slowdown)

Reply:
Performance should be unaffected, because I've added an update() call here, but I've also removed an update() call from Read206::readScore(), where it was unnecessary. I've run the profiler, and performance looks the same, with a total of 2 update() calls in both cases.

Reply:
Can you move this line to NotationProject::doLoad? It's very confusing to see it here

Reply:
done! 👍